src/components/cc-toggle/cc-toggle.js

```diff
@@ -130,7 +134,7 @@ export class CcToggle extends LitElement {
           ${repeat(this.choices, ({ value }) => value, ({ label, image, value }) => html`
             <input
               type=${type}
-              name="toggle"
+              name=${this.name ?? 'toggle'}
```

Comment:
I think you put a fallback to 'toggle' to avoid a breaking change.
I would prefer to introduce a breaking change and have a homogeneous way of handling name attributes on all our input components.

If you don't know what to do with this comment, let's wait for other reviewers to give their thoughts about that.